nvflare/fuel/utils/component_builder.py

```diff
@@ -46,6 +49,7 @@ def build_component(self, config_dict):
                     t = self.build_component(v)
                     class_args[k] = t
                 except BaseException:
+                    ComponentBuilder.logger.error(traceback.format_exc())
```

Comment:
Can we separate this to another PR?
Note that, this will print error for ALL config section that has a dict (because usually that dict is not a component)